src/io/inputs.jl

```diff
@@ -256,3 +227,48 @@ function _yaxarray2nameddimsarray(yarray::YAXArray)::NamedDimsArray
 
     return NamedDimsArray(data; zip(dim_names, dim_labels)...)
 end
+
+"""
+    DataCube(data::AbstractArray; kwargs...)::YAXArray
+
+Constructor for YAXArray.
+"""
+function DataCube(data::AbstractArray; kwargs...)::YAXArray
+    return YAXArray(Tuple(Dim{name}(kwargs[name]) for name in Symbol.(keys(kwargs))), data)
+end
+
+"""
+    ZeroDataCube(T=Float64; kwargs...)::YAXArray
+
+Constructor for YAXArray with all entries equal zero.
+"""
+function ZeroDataCube(T=Float64; kwargs...)::YAXArray
+    return DataCube(
+        zeros(T, Tuple(length(kwargs[name]) for name in Symbol.(keys(kwargs))));
```

Comment:
I've just changed the name of the YAXArray constructor (to avoid conflicts) so DataCube is the name of the constructor now. This creates a DataCube with the kwargs passed AND an empty array as `data`.